shared/profile/edit-avatar/index.js.flow

```diff
@@ -1,10 +1,11 @@
 // @flow
 import {Component} from 'react'
+import * as RPCTypes from '../../constants/types/rpc-gen'
 
 export type Props = {
-  keybaseUsername: string,
-  hasAvatar: boolean,
-  onAck: () => void,
+  image: Object,
```

Comment:
why not type this

Reply:
It's a prop that's only used on native, and I wasn't sure how to do that since this is shared with desktop. How do I do that? Is there a better way to do this?

Reply:
you can do ?{x: number, y: number, /etc) whatever it is, and its ok for it to be null

Reply:
When I do that, Flow complains:

```
Cannot get `this.props.image.uri` because property `uri` is missing in null or undefined [1].
```

Do I need to check for `this.props.image`'s existence every time I want to access its properties?

Reply:
yes cause thats what you're saying, it can be null sometimes